bitnami/mongodb/templates/common-scripts-cm.yaml

```diff
@@ -71,7 +71,11 @@ data:
     [alt_names]
     DNS.1 = $svc
     DNS.2 = $my_hostname
+    {{- if eq .Values.architecture "replicaset" }}
     DNS.3 = $my_hostname.$svc.$MY_POD_NAMESPACE.svc.cluster.local
+    {{- else }}
+    DNS.3 = $svc.$MY_POD_NAMESPACE.svc.cluster.local
+    {{- end }}
```

Comment:
No FDQN is required in standalone mode, as no headless service is generated.

I replaced DNS.3 with the long service name.